tests/test_fastavro.py

```diff
@@ -84,21 +84,21 @@ def test_not_avro():
 
 def test_acquaint_schema_rejects_undleclared_name():
     try:
-        fastavro.schema.acquaint_schema({
+        fastavro._schema.acquaint_schema({
```

Comment:
Did something change that makes the old imports from `fastavro.schema` not work anymore?

Reply:
I'm looking into this. A few tests fail without these changes. My hypothesis is that `schema.py` and `_schema.pyx` have mutable global state, and the breaking tests may be accidentally writing to one and reading from the other. Someone who knows this area of the code may have ideas here.